.gitignore

```diff
@@ -32,3 +32,5 @@ custom_cogs/*
 settings/*
 !settings/*.sample
 cogs/utils/temp.txt
+tmp/*
```

Comment:
As far as I can see, the bot does not create a `tmp` folder during normal operations...

Reply:
It used to create it in the `run.bat` and `run_linuxmac.sh` scripts as backups of the settings folder right before an update started. It has been renamed to `settings_backup` now.